boards/arm/mm_swiftio/mmswiftio/mmswiftio_flexspi_nor_config.c

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
why is there an `mmswiftio` directory under `mm_swiftio`? This extra layer seems unnecessary.

Reply:
Thanks for your review.
mmswiftio folder be used to place the boot header config file. In the future other spi nor and sdram may be used in mm_swiftio, we hope these config files be put in the folder for easy management

Reply:
Do you know for sure that you will support other memories on this board? If yes, then you need to come up with a more descriptive name for the subdirectory that suggests what memory configuration it is for. Otherwise, I recommend removing the extra layer now and adding it later if/when it's actually needed.

Reply:
Add other memories is a longer-term plan. I remove the extra layer as your suggestion. Thanks!